packages/snap-tracker/src/Tracker.ts

```diff
@@ -245,22 +245,23 @@ export class Tracker {
 					},
 				});
 			}
-
-			const { userAgent, href, filename, stack, message, colno, lineno, errortimestamp } = data;
+			const { useragent, href, filename, stack, message, colno, lineno, errortimestamp, details } = data;
 
 			const payload = {
 				type: BeaconType.ERROR,
 				category: BeaconCategory.RUNTIME,
 				context,
 				event: {
-					userAgent: userAgent || navigator.userAgent,
+					useragent: useragent || window?.navigator?.userAgent,
```

Comment:
I forgot to mention in previous comments - but we can get rid of useragent altogether here, and we should, to prevent confusion.

We are now getting the user agent details from the beacon request themselves (on the beacon side of things).